tests/integration/security/authorization_test.go

```diff
@@ -64,44 +61,46 @@ func TestAuthorization_mTLS(t *testing.T) {
 					"Namespace2": apps.Namespace2.Name(),
 					"dst":        dst[0].Config().Service,
 				}
-				policies := tmpl.EvaluateAllOrFail(t, args,
-					file.AsStringOrFail(t, "testdata/authz/v1beta1-mtls.yaml.tmpl"))
-				t.ConfigIstio().ApplyYAMLOrFail(t, apps.Namespace1.Name(), policies...)
-				t.ConfigIstio().WaitForConfigOrFail(t, t, apps.Namespace1.Name(), policies...)
+				config.WithWait("testdata/authz/v1beta1-mtls.yaml.tmpl")(t, apps.Namespace1.Name(), args)
+				callCount := 1
+				if dst.Clusters().IsMulticluster() {
+					// so we can validate all clusters are hit
+					callCount = util.CallsPerCluster * len(dst.Clusters())
```

Comment:
nit: this 4 lines is in a lot of places, could just make it abstracted or even automatic

Reply:
Yeah I think that's something we should simply consider within the echo framework somehow. I'm punting on it for now.